vscode/src/prompt-builder/unique-context.ts

```diff
@@ -105,3 +113,10 @@ function rangesOnSameLines(range1: RangeData, range2: RangeData): boolean {
 function isUserAddedItem(item: ContextItem): boolean {
     return getContextItemTokenUsageType(item) === 'user'
 }
+
+/**
+ * Checks if content is set and equal.
+ */
+function equalContent(item1: ContextItem, item2: ContextItem): boolean {
+    return !!item1.content && item1.content === item2.content
```

Comment:
```suggestion
    return !!item1.content && item1.content === item2.content
```

QQ: should we compare the trimmed content for cases where the differences are new lines etc?

Reply:
My initial reaction is to not treat whitespace as not significant. In particular I think about languages like python. However, in this case we are talking about the full file, so leading and trailing whitespace does indeed seem not significant. So I will update this check to match your suggestion.